dbms/src/Flash/ResourceControl/TokenBucket.h

```diff
@@ -95,6 +95,19 @@ class TokenBucket final
         return fmt::format("tokens: {}, fill_rate: {}, capacity: {}", tokens, fill_rate, capacity);
     }
 
+    uint64_t estWaitDuraMS(uint64_t max_wait_dura_ms) const
+    {
+        const auto tokens = peek();
+        if (tokens > 0 || fill_rate_ms == 0.0)
```

Comment:
```suggestion
        if (tokens >= 0 || fill_rate_ms == 0.0)
```

Reply:
If `tokens` is zero, return value will be `100`. But still changed, maybe it's more clear